conf/mesos-site.xml

```diff
@@ -49,7 +49,7 @@
 
   <property>
     <name>mesos.hdfs.cluster.name</name>
```

Comment:
Looks like SchedulerConf was changed to reference framework.name, but this still says cluster.name. Fix?